thirdweb/erc721_signature_minting.go

```diff
@@ -53,16 +53,26 @@ func newERC721SignatureMinting(provider *ethclient.Client, address common.Addres
 //	// Learn more about how to craft a payload in the Generate() function
 //	signedPayload, err := contract.Signature.Generate(payload)
 //	tx, err := contract.Signature.Mint(context.Background(), signedPayload)
-func (signature *ERC721SignatureMinting) Mint(ctx context.Context, signedPayload *SignedPayload721) (*types.Transaction, error) {
-	message, err := signature.mapPayloadToContractStruct(ctx, signedPayload.Payload)
+func (signature *ERC721SignatureMinting) MintAndAwait(ctx context.Context, signedPayload *SignedPayload721) (*types.Transaction, error) {
```

Comment:
Any particular reason to rename this here? If possible, would probably be best to keep the `Mint` API here available just so it isn't a breaking change

Reply:
as a sdk.. we should be really explicit and obvious when we are doing loop/poll .. hence the suffix

Reply:
sent another pr then: https://github.com/thirdweb-dev/go-sdk/pull/118